src/main/java/dev/meyi/bn/config/ModuleConfig.java

```diff
@@ -0,0 +1,17 @@
+package dev.meyi.bn.config;
```

Comment:
```suggestion
package dev.meyi.bn.json.config;
```

Let's move it here (since it's effectively a data class). the config directory still needs to be created though.